crates/daphne/src/hpke.rs

```diff
@@ -325,7 +325,7 @@ impl HpkeDecrypter for HpkeReceiverConfig {
 pub struct HpkeReceiverConfig {
     pub config: HpkeConfig,
     #[serde(with = "HpkePrivateKeySerde")]
-    private_key: HpkePrivateKey,
+    pub private_key: HpkePrivateKey,
```

Comment:
Why make this pub? Who needs access?

Reply:
Serialization code needs access to it. There's also no invariant the type is maintaining that requires this to be private